README.md

```diff
@@ -96,6 +96,31 @@ teams:
       project-groups:
         - <project group>
         - <project group>
+
+    # (Optional) Gitea configuration for the team
+    gitea:
+      # (Required) The name of a Gitea Organisation
+      - org: <gitea organisation name>
+
+        # (Required) The URL of the Gitea instance, e.g. `https://opendev.org/`
+        url: <URL for the Gitea instance>
+
+        # (Optional) A list of repository names to include
+        # If empty, all public, non-archived repositories in the org will be
+        # included, unless listed in `ignores`.
+        includes:
```

Comment:
Yeh I think this is a good default behaviour - we'll probably want to port this back to Launchpad/Github too so the forge's are as consistent as possible